src/js/stores/style.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2015 Adobe Systems Incorporated. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+define(function (require, exports, module) {
+    "use strict";
+
+    var Fluxxor = require("fluxxor");
+
+    var events = require("../events");
+
+    /**
+     * Manages a style clipboard and style link maps of active documents
+     */
+    var StyleStore = Fluxxor.createStore({
+        /**
+         * Currently stored style object for copy/pasting
+         * 
+         * @private
+         * @type {Object}
+         */
+        _storedStyle: null,
+
+        /** 
+         * Binds flux actions.
+         */
+        initialize: function () {
+            this._storedStyle = {
+                effects: {
+                    innerShadows: [],
+                    dropShadows: []
+                },
+                fillColor: null,
+                strokeColor: null,
+                typeStyle: null
+            };
+
+            this.bindActions(
+                events.style.COPY_STYLE, this._copyStyle
+            );
+        },
+
+        /**
+         * Returns the currently copied style object
+         *
+         * @return {Object}
```

Comment:
`object` when it's being referred to as a basic type instead of something more structured.